src/component/todo/todoItem.js

```diff
@@ -5,15 +5,13 @@ class Todoitem extends Component {
         super()
 
         this.state = {
-            task: '',
             status: false,
         }
     }
    
     checkItem = () => {
-       const newTask = this.state.status ? 'unActive' : 'isActive'
         this.setState({
-            task: newTask
+            status: true
```

Comment:
Status will be true forever when clicked. 
fix it, let 'status' can be changed when you clicked every time.